onboarding/source/controller.c

```diff
@@ -85,9 +89,24 @@ obc_error_code_t initController(void) {
         return OBC_ERR_CODE_TIMER_CREATION_FAILED;
 
     /* USER CODE BEGIN */
-    // Create light timer and check if it was created successfully
+    if (lightTimerHandle == NULL) {
+        // Create light timer
+        lightTimerHandle = xTimerCreateStatic(LIGHT_CONTROLLER_NAME,
+                                        LIGHT_CONTROLLER_PERIOD, 
+                                        LIGHT_CONTROLLER_AUTORELOAD,
+                                        (void *) 0,
+                                        lightTimerCallback,
+                                        &lightTimerBuffer);
+    }
+                                        
+    if (lightTimerHandle == NULL) 
+            return OBC_ERR_CODE_TIMER_CREATION_FAILED;
```

Comment:
It looks like the indentation is off here too. There's probably some setting in your IDE that you should change. Indentation should be 4 spaces